File: src/main/java/club/deneb/client/command/commands/Help.java
package club.deneb.client.command.commands;

import club.deneb.client.features.ModuleManager;
import club.deneb.client.Deneb;
import club.deneb.client.command.Command;
import club.deneb.client.command.CommandManager;
import club.deneb.client.utils.ChatUtil;
import org.lwjgl.input.Keyboard;

/**
 * Created by B_312 on 01/15/21
 */
@Command.Info(command = "help", description = "Get helps.")
public class Help extends Command {

    @Override
    public void onCall(String s, String[] args) {
        ChatUtil.printChatMessage("\2476" + Deneb.MOD_NAME + " " + "\247a" + Deneb.VERSION);
        ChatUtil.printChatMessage("\247c" + "Made by: " + Deneb.AUTHOR);
        ChatUtil.printChatMessage("\247c" + "Github: " + Deneb.GITHUB);
        ChatUtil.printChatMessage("\247a" + "Press " + "\247c" + Keyboard.getKeyName(ModuleManager.getModuleByName("ClickGUI").getBind()) + "\247a" + " to open ClickGUI");
        ChatUtil.printChatMessage("\247a" + "Press " + "\247c" + Keyboard.getKeyName(ModuleManager.getModuleByName("HUDEditor").getBind()) + "\247a" + " to open HUDEditor");
        ChatUtil.printChatMessage("\247a" + "Use command: " + "\247e" + CommandManager.INSTANCE.cmdPrefix + "prefix <target prefix>" + "\247a" + " to set command prefix");
        ChatUtil.printChatMessage("\247a" + "List all available commands: " + "\247e" + CommandManager.INSTANCE.cmdPrefix + "commands");
    }

    @Override
    public String getSyntax() {
        return "help";
    }

}

File: src/main/java/club/deneb/client/command/commands/Help.kt
package club.deneb.client.command.commands

import club.deneb.client.Deneb
import club.deneb.client.command.Command
import club.deneb.client.command.CommandManager
import club.deneb.client.features.ModuleManager
import club.deneb.client.utils.ChatUtil.printChatMessage
import org.lwjgl.input.Keyboard

/**
 * Created by B_312 on 01/15/21
 */
@Command.Info(command = "help", description = "Get helps.")
class Help : Command() {
    override fun onCall(s: String, vararg args: String) {
        printChatMessage("\u00a76" + Deneb.MOD_NAME + " " + "\u00a7a" + Deneb.VERSION)
        printChatMessage("\u00a7c" + "Made by: " + Deneb.AUTHOR)
        printChatMessage("\u00a7c" + "Github: " + Deneb.GITHUB)
        printChatMessage("\u00a7a" + "Press " + "\u00a7c" + Keyboard.getKeyName(ModuleManager.getModuleByName("ClickGUI").bind) + "\u00a7a" + " to open ClickGUI")
        printChatMessage("\u00a7a" + "Press " + "\u00a7c" + Keyboard.getKeyName(ModuleManager.getModuleByName("HUDEditor").bind) + "\u00a7a" + " to open HUDEditor")
        printChatMessage("\u00a7a" + "Use command: " + "\u00a7e" + CommandManager.INSTANCE.cmdPrefix + "prefix <target prefix>" + "\u00a7a" + " to set command prefix")
        printChatMessage("\u00a7a" + "List all available commands: " + "\u00a7e" + CommandManager.INSTANCE.cmdPrefix + "commands")
    }

    override fun getSyntax(): String {
        return "help"
    }
}